charts/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: prometheus
-version: 13.0.2
```

Comment:
@bismarck  cool, thanks for the feedback.  I just keep bumping the version to avoid conflit because the PR is almost a month old.. I didn't have conflit back then.  

Just tell me which version I should put there and I'll do it

Reply:
@survivant just revert the changes to this file. It isn't relevant to your PR.

Reply:
should be ok @bismarck